Unit-01/04-flask-crud/app.py

```diff
@@ -1,3 +1,57 @@
-from flask import Flask
+from flask import Flask, request, render_template, url_for, redirect
+from snack import Snack
+from flask_modus import Modus
 
-snack_list = []
\ No newline at end of file
+snack_list=[]
+
+app = Flask(__name__)
+modus = Modus(app)
+
+def find_snack(id):
+	return [snack for snack in snack_list if snack.id == id]
```

Comment:
it might be a better idea to take the first element here instead of later since the function is called `find_snack`. Alternatively, you could rename this `find_snacks` since it's multiple